.github/workflows/before_merge.yaml

```
@@ -93,8 +93,52 @@ jobs:
           name: model-tests-metrics-group-${{ matrix.group }}
           path: metrics/
 
+  push-autogen-op-tests:
+    needs: [model-tests]
+    if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.commit_report != 'None'}}
```

```suggestion
    if: ${{ github.event_name == 'workflow_dispatch' && (github.event.inputs.commit_report == 'All' || github.event.inputs.commit_report == 'Tests'}}
```

This will allow to update Docs without regenerating all tests, and will run them in parallel with models, right?

---

also here
```
        description: 'Commit generated report files: None, Docs, All'
```
must update to
```
        description: 'Commit generated report files: None, Docs, Tests, All'
```